SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1386,18 +1386,13 @@ void USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConne
 				}
 			}
 
-			// UNR-865 - Handle closing actor channels for non-relevant actors without deleting the entity.
-			// If the actor wasn't recently relevant, or if it was torn off, close the actor channel if it exists for this connection
+			// If the actor has been torn off, close the channel
+			// Native also checks here for !bIsRecentlyRelevant and if so closes due to relevancy, we're not doing because it's less likely
+			// in a SpatialOS game. Might be worth an investigation in future as a performance win.
```

Comment:
Let's get a ticket for this

Reply:
👌

Reply:
This seems like one of those tickets that's going to sit in the backlog and then get closed after 6 months :P

Reply:
There was already a ticket... Not 6 months old just yet...